.github/workflows/make.yml

```diff
@@ -18,7 +18,11 @@ jobs:
         python-version: 3.7
     - name: configure
       run: sudo apt install expect && pip3 install python-chess && git clone https://github.com/ddugovic/chess-artist.git ./tests/chess-artist --depth 1
-    - name: make
+    - name: make default
```

Comment:
I'm not excited about deviating too much from https://github.com/official-stockfish/Stockfish/tree/master/.github/workflows

Reply:
I mean this Pr allows to download latest Windows Binaries from workflow example like 
https://github.com/SriMethan/Stockfish/actions/runs/1265045451 https://github.com/SriMethan/Stockfish/suites/3850374647/artifacts/95544413

Reply:
I guess I'll first sync my repository with `official-stockfish/master` and then see what the differences are.

Reply:
Ok so should I close this pr or just keep it?

Reply:
I guess keep it, there's no hurry.

Reply:
I merged `official-stockfish/master` into my branch.  If you can fix merge conflicts and if the build succeeds I'll accept this PR.

Reply:
Ok I will send a new Pr for this

Reply:
@ddugovic this is the new pr https://github.com/ddugovic/Stockfish/pull/594